Adjust pixel tolerance for compositing/video/video-border-radius-clipping.html

karlrackler opened the pull request:
#### 096366b14a76fa4648dba3b1df9d63521fb6cfc2
<pre>
Adjust pixel tolerance for compositing/video/video-border-radius-clipping.html
<a href="https://bugs.webkit.org/show_bug.cgi?id=257646">https://bugs.webkit.org/show_bug.cgi?id=257646</a>
rdar://104152683

Reviewed by Simon Fraser.

This adds pixel tolerance for maxDifference and totalPixels as needed by actual result.

* LayoutTests/compositing/video/video-border-radius-clipping.html:

Canonical link: <a href="https://commits.webkit.org/264994@main">https://commits.webkit.org/264994@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/a770824ee7ab5236e9c0d55f53eacbce8c6b9710

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/9397 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/9674 "Built successfully") | [❌ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/9905 "Failed to checkout and rebase branch from PR 14626") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/11059 "Built successfully") | [  ~~🛠 wincairo~~](https://ews-build.webkit.org/#/builders/32/builds/9269 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/11661 "Built successfully") | [  ~~🛠 mac-AS-debug~~](https://ews-build.webkit.org/#/builders/16/builds/9636 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/12150 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/9548 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/15/builds/10462 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [❌ 🧪 api-mac](https://ews-build.webkit.org/#/builders/14/builds/9905 "Failed to checkout and rebase branch from PR 14626") | [  ~~🛠 gtk~~](https://ews-build.webkit.org/#/builders/2/builds/11216 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/39/builds/7759 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [❌ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/14/builds/9905 "Failed to checkout and rebase branch from PR 14626") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/2/builds/11216 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/8856 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [❌ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/14/builds/9905 "Failed to checkout and rebase branch from PR 14626") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/2/builds/11216 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/9220 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/16/builds/9636 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [  ~~🛠 tv-sim~~](https://ews-build.webkit.org/#/builders/30/builds/8432 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [❌ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/14/builds/9905 "Failed to checkout and rebase branch from PR 14626") | | 
| [  ~~🛠 🧪 merge~~](https://ews-build.webkit.org/#/builders/19/builds/2274 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/12657 "Built successfully") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/1074 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/8982 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

smfr commented on LayoutTests/compositing/video/video-border-radius-clipping.html:
```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <html>
 <head>
+    <meta name="fuzzy" content="maxDifference=0-42; totalPixels=0-197" />
```
No need for the `/>` at the end; just use `>`

karlrackler committed:
Adjust pixel tolerance for compositing/video/video-border-radius-clipping.html
https://bugs.webkit.org/show_bug.cgi?id=257646
rdar://104152683

Reviewed by Simon Fraser.

This adds pixel tolerance for maxDifference and totalPixels as needed by actual result.

* LayoutTests/compositing/video/video-border-radius-clipping.html:

Canonical link: https://commits.webkit.org/264994@main
